imports/plugins/unit-test/remote-graphql/server/no-meteor/schemas.js

```diff
@@ -0,0 +1,14 @@
+import {
+  makeExecutableSchema,
+  makeRemoteExecutableSchema
+} from "graphql-tools";
+import { HttpLink } from "apollo-link-http";
+import fetch from "node-fetch";
+import { PORT } from "../../../../../../tests/mocks/mockRemoteGraphql";
+
+const graphqlUrl = `http://localhost:${PORT}/graphql`;
```

Comment:
The hard coded port and the extra complexity makes me nervous. Did you try just mocking the HTTP request with `nock` package? It is already a dependency.

We already have random issues with findFreePort when running on CI, so I'm planning to update `TestApp` to use https://www.apollographql.com/docs/apollo-server/features/testing.html#createTestClient rather than starting an actual server.

Reply:
OK switched to `nock` and it's working fine. Awesome change. I :heart: `nock`.